packages/interface/src/__tests__/getDiagnostics-test.js

```diff
@@ -47,6 +47,41 @@ describe('getDiagnostics', () => {
     expect(error.source).to.equal('GraphQL: Deprecation');
   });
 
+  it('returns no errors for valid query', () => {
+    const errors = getDiagnostics('query { hero { name } }', schema);
+    expect(errors.length).to.equal(0);
+  });
+
+  it('returns no errors for valid query', () => {
+    const errors = getDiagnostics('query { hero { name } }', schema);
+    expect(errors.length).to.equal(0);
+  });
+
+  it('returns no errors for valid query with aliases', () => {
+    const errors = getDiagnostics(
+      'query { superHero: hero { superName :name } }',
+      schema,
+    );
+    expect(errors.length).to.equal(0);
+  });
+
+  it('catches a syntax error in the SDL', () => {
+    const errors = getDiagnostics(
+      `
+        type Human implements Character {
+          SyntaxError
```

Comment:
wat, not sure I understand `SyntaxError` as a field - maybe I'm not caught up with the current GraphQL spec. Could you elaborate here?

Reply:
Oops! my bad, I wanted to show that this is a syntax error i.e. type name without a named type definition. Using camel case was wrong, changing it.

Reply:
Used a verbose `field_without_type_is_a_syntax_error` for field name. Please let me know if I should change it :)